.ci/jobs/elastic+helm-charts+master+template-lint-python.yml

```diff
@@ -0,0 +1,20 @@
+---
+- job:
+    name: elastic+helm-charts+master+lint-python
+    display-name: elastic / helm-charts - master - lint python
+    description: Master - lint python
+    scm:
+    - git:
+        wipe-workspace: 'True'
+    axes:
+    - axis:
+        type: slave
+        name: label
+        values:
+        - docker&&virtual
+    builders:
+    - shell: |-
+        #!/usr/local/bin/runbld
+        set -euo pipefail
+
+        make lint-python
```

Comment:
Do the python dependencies need installed before this? Otherwise `black` might not be available I think.

Reply:
nice catch, fixed in https://github.com/elastic/helm-charts/pull/475/commits/8e0c1698a29f64fa5427db6d20ac976ddbb3d50f